boards/cats_rev1Pro/src/cli/cli.c

```diff
@@ -820,6 +968,11 @@ static void processCharacter(const char c) {
 }
 
 static void processCharacterInteractive(const char c) {
+  static uint16_t ignore = 0;
+  if (ignore) {
+    ignore--;
+    return;
+  }
```

Comment:
What is the purpose of this? Pls add a comment in code

Reply:
and describe here

Reply:
It's to get rid of the up arrow characters, the up arrow prints 3 bytes of data if I remember correctly, I only look at the first byte

Reply:
ok, pls add a comment about this.